### PR TITLE
Wrapper shell scripts for starting and stopping services

### PR DESCRIPTION
**Pull request by getplank**
Start and stop scripts (for *nix environments) to automate the instructions provided at https://github.com/bitpay/bitcore-wallet-service/blob/master/installation.md

If this is accepted, I can update the documentation as well.

### CODE REVIEW

**Comment by matiu** [comment]
Yes! thanks for the contribution!

Could you update the docs and also add 'start.sh' to package.json ->  npm start?

**Comment by getplank** [comment]
Done. Also added a "stop" target to package.json

All doc changes in installation.md. Cleaned up some inaccurate npm references there as well. The installation document is completely accurate and tested.

File: installation.md [hunk 1]
```diff
@@ -1,15 +1,42 @@
...
+npm start
+```
+### Configuration
+Configuration for all required modules can be specified in https://github.com/bitpay/bitcore-wallet-service/blob/master/config.js
 
-###Start MongoDB
+BWS is composed of 5 separate node services -
+Locker - locker/locker.js
+Message Broker - messagebroker/messagebroker.js
+Blockchain Monitor - bcmonitor/bcmonitor.js (This service talks to the Insight service hosted by Bitpay)
```
Comment (isocolsky) [inline installation.md]: Actually, `bcmonitor` will use whatever instance is specified in `config.js` under `blockchainExplorerOpts`

Comment (getplank) [inline installation.md]: corrected

**Comment by isocolsky** [comment]
LGTM
This is very useful, thank you!

**Pull request merged**